specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -2495,31 +2498,31 @@
         },
         "rdmaNetworkInterfaceCount": {
           "$ref": "#/definitions/VMAttributeMinMaxInteger",
-          "description": "The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+          "description": "The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.\nrdmaSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf rdmaSupport is \"Excluded\", this VMAttribute can not be used."
         },
         "acceleratorSupport": {
           "$ref": "#/definitions/VMAttributeSupport",
-          "description": "Specifies whether the VMSize supporting accelerator should be used to build Fleet or not."
+          "description": "Specifies whether the VMSize supporting accelerator should be used to build Fleet or not.\nacceleratorSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf acceleratorSupport is \"Excluded\", this VMAttribute can not be used."
         },
         "acceleratorManufacturers": {
           "type": "array",
-          "description": "The accelerator manufacturers specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.",
+          "description": "The accelerator manufacturers specified as a list. \nacceleratorSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf acceleratorSupport is \"Excluded\", this VMAttribute can not be used.",
           "items": {
             "$ref": "#/definitions/AcceleratorManufacturer"
           },
           "x-ms-identifiers": []
         },
         "acceleratorTypes": {
           "type": "array",
-          "description": "The accelerator types specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.",
+          "description": "The accelerator types specified as a list. acceleratorSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf acceleratorSupport is \"Excluded\", this VMAttribute can not be used.",
           "items": {
             "$ref": "#/definitions/AcceleratorType"
           },
           "x-ms-identifiers": []
         },
         "acceleratorCount": {
           "$ref": "#/definitions/VMAttributeMinMaxInteger",
-          "description": "The range of accelerator count specified from min to max.. Optional parameter. Either Min or Max is required if specified."
+          "description": "The range of accelerator count specified from min to max. Optional parameter. Either Min or Max is required if specified.\nacceleratorSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf acceleratorSupport is \"Excluded\", this VMAttribute can not be used."
         },
         "vmCategories": {
```

Comment:
Looks like we missed adding the following VMAttribute. This was a part of the API spec I shared. memoryInGiB and memoryInGiBPerVCpu are 2 different attributes. I see only memoryInGiB in this file.

/// <summary>
/// Optional.
/// The range of memory in GiB per vCPU specified from min to max.
/// Either min or max is required if specified.
/// </summary>
public VMAttributeMinMaxDouble memoryInGiBPerVCpu;

Reply:
Ok, I will add it, it was removed as per comment.